components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineView.kt

```diff
@@ -65,6 +66,12 @@ interface EngineView {
     * true if can and false otherwise.
     */
     fun canScrollVerticallyDown(): Boolean = true
+
+    /**
+     * Take a screenshot of the actual session.
+     * @param onFinish A callback to inform that process of capturing a thumbnail has finished.
+     */
```

Comment:
nit: Using the term `session` here may be confusing? Maybe we can borrow from the GeckoView docs: 
https://mozilla.github.io/geckoview/javadoc/mozilla-central/org/mozilla/geckoview/GeckoView.html#capturePixels--

Reply:
👍